elastica/rod/cosserat_rod.py

```diff
@@ -30,6 +30,91 @@ def _get_z_vector():
 
 
 class CosseratRod(RodBase):
+    """
+    Cosserat Rod class. This is the preferred class for rods because it is derived from some
+    of the essential base classes.
+    Although the attributes of this CosseratRod class are inherited from its parent classes, for convenience and easy access, the variable
+    names are given below.
```

Comment:
I think this statement is not accurate for the current code. Since `RodBase` does not have any attributes or methods (other than `__init__` )

Reply:
I agree with Arman. Maybe this explanation is not necessary. We don't have any other rod than CosseratRod at this point.

Reply:
I agree. I removed the relevant sentence.